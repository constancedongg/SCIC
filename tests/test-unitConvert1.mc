float '{cm} x;
float '{m} y;

int func main(){
    y = 1.022;
    x = y; 
    printf(x); 
    return 0;
}