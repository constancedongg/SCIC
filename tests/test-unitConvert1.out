0.001022
0.34237
